tensorflow_addons/metrics/cohens_kappa.py

```diff
@@ -38,28 +38,30 @@ class CohenKappa(Metric):
 
     Usage:
 
-    ```python
-    actuals = np.array([4, 4, 3, 4, 2, 4, 1, 1], dtype=np.int32)
-    preds = np.array([4, 4, 3, 4, 4, 2, 1, 1], dtype=np.int32)
-    weights = np.array([1, 1, 2, 5, 10, 2, 3, 3], dtype=np.int32)
 
-    m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
-    m.update_state(actuals, preds)
-    print('Final result: ', m.result().numpy()) # Result: 0.61904764
+    >>> actuals = np.array([4, 4, 3, 4, 2, 4, 1, 1], dtype=np.int32)
+    >>> preds = np.array([4, 4, 3, 4, 4, 2, 1, 1], dtype=np.int32)
+    >>> weights = np.array([1, 1, 2, 5, 10, 2, 3, 3], dtype=np.int32)
 
-    # To use this with weights, sample_weight argument can be used.
-    m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
-    m.update_state(actuals, preds, sample_weight=weights)
-    print('Final result: ', m.result().numpy()) # Result: 0.37209308
-    ```
+    >>> m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
+    >>> # m.update_state(actuals, preds)
+    >>> # print('Final result: ', m.result().numpy())
+    >>> # Final result: 0.61904764
```

Comment:
Why these are commented out?

Reply:
please see the image below. 
I don't reckon what do you mean by 'all results from metric(y_true, y_pred) should be named result'. do you mean whenever a metric is directly called or do you mean metric.update_state(y,ypred), should be assigned to result?. if I assign update_state to result the print statement should be like
`print('Final result: ', result.result().numpy())` 
can you please check this [PR](https://github.com/tensorflow/addons/pull/2109). it has a similar issue as described below. I have not reverted back the changes 
![image](https://user-images.githubusercontent.com/5617255/92317247-38243600-f018-11ea-9647-42e3814fc4a6.png)